package.json

```diff
@@ -90,7 +90,7 @@
     "request": "^2.81.0",
     "sass-loader": "^6.0.6",
     "sitemap-static": "^0.4.2",
-    "static-site-generator-webpack-plugin": "^3.4.1",
+    "static-site-generator-webpack-plugin": "git://github.com/EugeneHlushko/static-site-generator-webpack-plugin.git#957984e4fe7592f534b1949736bef7251b6e832c",
```

Comment:
I think using `git://` can lead to new issues. Maybe use `https://` or can we wait for a while if your PR gets merged? :)